dom.bs

```diff
@@ -6519,6 +6529,11 @@ string <var>namespace</var> (default null):</p>
 
  <li><p>If <var>oldAttr</var> is <var>attr</var>, return <var>attr</var>.
 
+ <li><p>Let <var>verifiedValue</var> be the result of calling <a>verify attribute value</a>
```

Comment:
This covers, setAttributeNode, setAttributeNodeNS, setNamedItem and setNamedItemNS